### ISSUE TITLE
Assigning version and appVersion to kiali-server helm charts

### ISSUE THREAD

**Issue by ethernoy** [kiali/kiali#3993]
**Is your feature request related to a problem? Please describe.**
It is not that convenient to manage kiali-server when there are multiple versions of this chart available in a repository and all of them have same version number 0.0.0.

**Describe the solution you'd like**
Assign a chart version to kiali-server helm chart.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by jmazzitelli** [comment]
Can you explain? I see the correct versions: https://kiali.org/helm-charts/index.yaml

**Comment by ethernoy** [comment]
> Can you explain? I see the correct versions: https://kiali.org/helm-charts/index.yaml

Oh thx, I was thinking that the source of Kiali helm charts should be the git repository https://github.com/kiali/helm-charts. Seems I should refer to https://kiali.org/helm-charts instead.

**Comment by jmazzitelli** [comment]
Right. [The source of the Chart.yaml](https://github.com/kiali/helm-charts/blob/v1.34.0/kiali-server/Chart.yaml#L4-L5) is not the source of truth of the published helm charts - [it is the published metadata](https://kiali.org/helm-charts/index.yaml). That's merely the source yaml (which rarely needs to change) from which that published metadata are derived. When the helm charts are published, the version/appVersion are defined then - see: https://github.com/kiali/helm-charts/blob/v1.34.0/Makefile#L67 where the `helm package` command is told what the version/appVersion is [via the appropriate command line options](https://helm.sh/docs/helm/helm_package/#options).
